packages/fetchai/connections/soef/connection.py

```diff
@@ -56,9 +56,57 @@
 PUBLIC_ID = PublicId.from_str("fetchai/soef:0.3.0")
 
 
+NOT_SPECIFIED = object()
+
+PERSONALITY_PIECES = [
```

Comment:
Better to call it `PERSONALITY_PIECES_KEYS` imo

Reply:
done